Finish the club teaser

dPliakos opened the pull request:
- [x] Remove remaining teaser box classes, 
- [x] check distances, 
- [x] add white border to image

Fix #33

iropal commented:
@dPliakos You still need to fix a few of things:
1. Top and bottom distance need 5px extra distance each. I would add an internal wrapper with the extra paddings.
2. .club-teaser__text needs 8px padding top.
3. We'd better overwrite the title font size and line height with the ones in the design. I would propose including typeface title 
4. .club-teaser__image-wrapper box shadow should be 0px 0px 1px 1px rgba(0, 0, 0, 0.1)

I think tats all ;)

dPliakos commented:
I fixed the distances.
The `typeface('title')` does not add the `letter-spacing` and `font-size`, I added them manually.
The change at the `box-shadow` was intended. To match the second border at the design. I changed it.
